manifests/supervisorcluster/1.20/cns-csi.yaml

```diff
@@ -104,6 +104,34 @@ subjects:
     kind: Group
     name: system:serviceaccounts:vmware-system-csi
 ---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: vsphere-admin-csi-role
+rules:
+  - apiGroups: ["cns.vmware.com"]
+    resources: ["cnsregistervolumes"]
+    verbs: ["get", "list", "create", "delete", "watch"]
```

Comment:
Can you explain why we need to have `delete` and `watch` privileges when we are trying to migrate volumes from TKGm to TKGs?

Reply:
`delete` required in case of rollback action when import of volume fails.

Reply:
`delete` required in case of rollback action when import of volume fails.
`watch` required to have informers on this object to check when the status is succeeded.